### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by hezhizhen**
Fixed some typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -709,7 +709,7 @@ We then suggest you add our one-liner to your shell `.rc` and a symlink
 for `/usr/local/bin/tea`.
 
 We might not have installed tea, if you used `sh <(curl tea.xyz) foo` and tea
-wasn’t already installed, then you we only fetched any packages, including
+wasn’t already installed, then you only fetched any packages, including
```
Comment (mxcl) [inline README.md:712]: actually I think `we` is correct.

**Comment by mxcl** [comment]
Good changes, bar one I think.

File: README.md (hunk 1 above)
Comment (hezhizhen) [reply README.md]: Done. PTAL.

**Review by mxcl — APPROVED** [review]
Thank you

**Pull request merged**